docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -216,7 +216,8 @@ HTTP retry :ref:`architecture overview <arch_overview_http_routing_retry>`.
 
 retry_on
   *(required, string)* specifies the conditions under which retry takes place. These are the same
-  conditions documented for :ref:`config_http_filters_router_x-envoy-retry-on`.
+  conditions documented for :ref:`config_http_filters_router_x-envoy-retry-on` and
```

Comment:
It seems a bit odd to me that in the route, we still HTTP/gRPC retry policies into a single field, but we have 2 different headers. I don't really feel strongly about this, but thought I would mention it in case anyone else has any strong opinions.